source/less/blocks/platforms-list.less

```
@@ -2,4 +2,36 @@
   margin: 0;
   padding: 0;
   list-style: none;
+
+  width: 205px;
+  padding-top: 35px;
+  padding-bottom: 21px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.platforms-list__link {
+  display: block;
+  width: 45px;
+  height: 55px;
+  background-repeat: no-repeat;
+
+  &--apple {
+    background-image: url("../img/svg/apple-logo.svg");
```

```suggestion
    background-image: url("/img/svg/apple-logo.svg");
```
абсолютные урлы.
Тут принцип простой - есть сервер - урлы абсолютные. А сервер есть всегда.

---

поправить для всех